tests/cross_functional/system_test/test_clone_deletion_after_max_cluster_space_utilization.py

```diff
@@ -0,0 +1,214 @@
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.utility.prometheus import PrometheusAPI
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_external_mode,
+    magenta_squad,
+    system_test,
+    polarion_id,
+)
+from ocs_ci.ocs.cluster import (
+    change_ceph_full_ratio,
+    CephCluster,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@system_test
+@polarion_id("OCS-5763")
+@pytest.mark.parametrize(
+    argnames=["interface_type"],
+    argvalues=[
+        pytest.param(constants.CEPHBLOCKPOOL),
+        pytest.param(constants.CEPHFILESYSTEM),
+    ],
+)
+class TestCloneDeletion(E2ETest):
+    """
+    Tests to verify clone deletion without error
+    after cluster out of full ratio
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, interface_type, pvc_factory, pod_factory, request):
+        """
+        create resources for the test
+
+        Args:
+        interface_type(str): The type of the interface
+            (e.g. CephBlockPool, CephFileSystem)
+        pvc_factory: A fixture to create new pvc
+        pod_factory: A fixture to create new pod
+
+        """
+
+        def teardown():
+            # change ceph full ratio to standard value
+            change_ceph_full_ratio(85)
+
+        request.addfinalizer(teardown)
+
+        logger.info("Starting the test setup")
+
+        self.num_of_clones = 25
+
+        # Getting the total Storage capacity
+        self.ceph_cluster = CephCluster()
+        self.ceph_capacity = int(self.ceph_cluster.get_ceph_capacity())
+        logger.info(f"ceph_capacity: {self.ceph_capacity}")
+
+        # Getting the free Storage capacity
+        self.ceph_free_capacity = int(self.ceph_cluster.get_ceph_free_capacity())
+        logger.info(f"ceph_free_capacity: {self.ceph_free_capacity}")
+
+        self.osd_full_size = int(self.ceph_capacity * 0.85)
+        logger.info(f"osd_full_size: {self.osd_full_size}")
+
+        self.currently_used_capacity = int(self.ceph_capacity - self.ceph_free_capacity)
+        logger.info(f"currently_used_capacity: {self.currently_used_capacity}")
+
+        # Available free storage capacity in the test
+        self.capacity_to_write = int(self.osd_full_size - self.currently_used_capacity)
+        logger.info(f"capacity_to_wrie: {self.capacity_to_write}")
+
+        # Calculating the file size
+        self.filesize = int(self.capacity_to_write / (self.num_of_clones + 1))
+
+        logger.info(f"filesize: {self.filesize}")
+
+        # Calculating the PVC size in GiB
+        self.pvc_size = int(self.filesize * 1.2)
+        logger.info(f"pvc size: {self.pvc_size}")
+
+        # Making filesize to MB
+        self.filesize = f"{int(self.filesize) * constants.GB2MB}M"
+
+        logger.info(
+            f"Total capacity size is : {self.ceph_capacity} GiB, "
+            f"Free capacity size is : {self.ceph_free_capacity} GiB, "
+            f"With {self.num_of_clones} clones to {self.pvc_size} GiB PVC. "
```

Comment:
With {self.num_of_clones} clones to {self.pvc_size} GiB PVC.
this message is not clear, please repharse it